### PR TITLE
Dzelge 973 hide secrets

### PR DESCRIPTION
**Pull request by dzelge**
When accepting this PR the cate op `read_zarr` will use the type `Password` for the bucket secret or token

### CODE REVIEW

File: cate/ops/io.py [hunk 1]
```diff
@@ -512,8 +512,8 @@ def write_geo_data_frame(gdf: gpd.GeoDataFrame,
 @op_input('drop_variables', data_type=VarNamesLike)
```
Comment (forman) [inline cate/ops/io.py:512]: That is the intended way

```suggestion
@op_input('drop_variables', data_type=VarNamesLike)
@op_input('secret', data_type=str, password=True)
@op_input('token', data_type=str, password=True)
```

File: cate/core/types.py [hunk 2]
```diff
@@ -157,6 +157,10 @@ def raise_validation_error(cls, value: Any):
 VarNames = List[str]
 
 
+class Password(str):
+    pass
+
+
```
Comment (forman) [inline cate/core/types.py:163]: Sorry, that is not the intended way.

File: CHANGES.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ## Version 2.1.5 (in development)
 
-* Now including ODP dataset verification information in data sources for use by Cate App. 
+* Now including ODP dataset verification information in data sources for use by Cate App.
+* read_zarr is now using type `Password` for the bucket secret or token
```
Comment (forman) [inline CHANGES.md:4]:
```suggestion
* Operation `read_zarr()` now uses the new `password` attribute its `secret`and `token` parameters.
```

**Review by forman — CHANGES_REQUESTED** [review]
Please use `password` attribute passed to `@op_input` decorator.

**Review by TonioF — CHANGES_REQUESTED** [review]
The entry in changes.md is not understandable. It seems like one word is missing.
Otherwise okay.

**Review by forman — APPROVED** [review]

**Review by forman — APPROVED** [review]

**Pull request merged**